text/0202-events-and-logs-api.md

```diff
@@ -0,0 +1,172 @@
+- include_trace_context (optional): Specifies whether the Trace Context should automatically be passed on to the events and logs created by the Logger. This SHOULD be false by default.
+
+It is unspecified whether or under which conditions the same or different Logger instances are returned from this function.
+
+Implementations MUST NOT require users to repeatedly obtain an Logger again with the same name+version+schema_url+event_domain+include_trace_context to pick up configuration changes. This can be achieved either by allowing to work with an outdated configuration or by ensuring that new configuration applies also to previously returned Loggers.
+
+Note: This could, for example, be implemented by storing any mutable configuration in the LoggerProvider and having Logger implementation objects have a reference to the LoggerProvider from which they were obtained. If configuration must be stored per-Logger (such as disabling a certain Logger), the Logger could, for example, do a look-up with its name+version+schema_url+event_domain+include_trace_context in a map in the LoggerProvider, or the LoggerProvider could maintain a registry of all returned Loggers and actively update their configuration if it changes.
+
+The effect of associating a Schema URL with a Logger MUST be that the telemetry emitted using the Logger will be associated with the Schema URL, provided that the emitted data format is capable of representing such association.
+
+### Logger
+
+The Logger is responsible for creating Events and Logs.
+
+Note that Loggers should not be responsible for configuration. This should be the responsibility of the LoggerProvider instead.
+
+#### Logger operations
+
+The Logger MUST provide functions to:
+
+- Create an `Event` and emit it to the processing pipeline.
+  - The API MUST accept an event name as a parameter. The event name provided should be inserted as an attribute with key `event.name`. Care MUST be taken to not override or delete this attribute while the `Event` is created.  This function MAY be named `logEvent`.
+- Create a `Log Record` and emit it to the processing pipeline.
+  - This function MAY be named `logRecord`.
+  - The intended users of this API is Log Appenders.
+
+### Usage
+
+```java
+OpenTelemetry openTelemetry = OpenTelemetry.noop();
+Logger logger = openTelemetry.getLogger("my-scope");
+
+// Using the convenience method to log an event directly
+logger.logEvent("network-changed", 
+                 Attributes.builder().put("type", "wifi").build());
+
+// Using the event builder to log an event
+logger.eventBuilder("page-navigated").build().setAttribute("url", "http://foo.com/bar#new").emit();
+
+// Using the logRecord builder to log a record
+Logger logger = openTelemetry.getLogger("another-scope");
+logger.logRecordBuilder().build().setBody("I am a log message").emit();
+
+```
+
+### Usage in Client-side telemetry
```

Comment:
I see the example code and it looks fine but again, I would have hoped to see a description here of when the event API should be used in client-side telemetry and the value provided. This again looks more like a draft spec to me than an OTEP

Reply:
> when the event API should be used in client-side telemetry and the value provided

It is for creating standalone events that occur when there is no span in progress. I updated the text with this.

Reply:
I don't think the example code is providing anything here. OTEPs should focus on motivations and leave implementation detail to the spec and SDKs.